moped-etl/arcgis/components_to_agol.py

```diff
@@ -115,6 +131,29 @@ def make_all_features(data):
                 attributes=component,
             )
             all_features["combined"].append(line_feature)
+
+            project_component_id = feature["attributes"]["project_component_id"]
+            # Filter exploded_geometry to only include dicts with matching project_component_id
+            matching_exploded_geometry_records = [
+                feature
+                for feature in exploded_geometry
+                if feature.get("project_component_id") == project_component_id
+            ]
+            for record in matching_exploded_geometry_records:
+                geometry = record.pop("geometry")
+                esri_geometry_key = "point"
+
+                feature = make_esri_feature(
+                    esri_geometry_key="point",
+                    geometry=geometry,
+                    attributes=record,
+                )
+
+                # it's unclear what this needs to be. 'point' seems the most correct.
+                # please see lines 117-118 for where the 'point' choice was drawn from.
```

Comment:
can you clarify this comment or remove it? i think the line references are broken.

Reply:
@frankhereford and I chatted about this in slack, and I think this can go altogether since the lines shifted and probably will again.

Reply:
It's outta there! a61579f63338c2db59d631f8f6f37124c25b968e